permissionless-batches/conf/coordinator/coordinator_run.sh

```
@@ -0,0 +1,127 @@
+  "https://circuit-release.s3.us-west-2.amazonaws.com/scroll-zkvm/releases/$SCROLL_ZKVM_VERSION/verifier/verifier.bin"
+  "https://circuit-release.s3.us-west-2.amazonaws.com/scroll-zkvm/releases/$SCROLL_ZKVM_VERSION/verifier/root-verifier-vm-config"
+  "https://circuit-release.s3.us-west-2.amazonaws.com/scroll-zkvm/releases/$SCROLL_ZKVM_VERSION/verifier/root-verifier-committed-exe"
+)
+
+# Function to download and verify files (skips existing valid files)
+download_and_verify() {
+  local url="$1"
+  local dest_dir="$2"
+  local checksum_file="$3"
+
+  local filename=$(basename "$url")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)